enterprise/frontend/src/embedding-sdk-bundle/components/public/dashboard/SdkDashboardStyleWrapper.tsx

```diff
@@ -13,7 +13,7 @@ export const SdkDashboardStyledWrapper = forwardRef(
   ) {
     return (
       <Flex
-        mih="100vh"
```

Comment:
I'm a bit afraid that this will break things. Could we verify in Shoppy and Embed Flow that the dashboard height does not change?

Reply:
@heypoom  I made it so it defaults to 100vh but can be overriden